test/e2e/clusterlogging_test.go

```diff
@@ -160,18 +162,17 @@ func clusterLoggingInitialDeploymentTest(t *testing.T, f *framework.Framework, c
 	if err != nil {
 		return err
 	}
+	err = WaitForElasticsearchResouce(t, f, namespace, retryInterval, timeout)
```

Comment:
Resource - and unrelated?

Reply:
will correct... and it is related because we only need to validate the CLO creates an ES resource.  We will defer additional e2e testing for the EO to that repo.